builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,29 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered metrics.Counter
+	buildsFailed    metrics.LabeledCounter
+)
+
+func init() {
```

Comment:
Maybe we should make constants for the various metrics labels, otherwise we could end up with a panic somewhere in the code due to typos, or accidentally add a new metric label without initializing it first.

Could potentially be in a separate package like `builder/internal/metrics`